spec/load_fixtures_helper.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper.rb'
+
+RSpec.configure do |rspec|
+  # This config option will be enabled by default on RSpec 4,
+  # but for reasons of backwards compatibility, you have to
+  # set it on RSpec 3.
+  #
+  # It causes the host group and examples to inherit metadata
+  # from the shared context.
+  rspec.shared_context_metadata_behavior = :apply_to_host_groups
+end
+
+RSpec.shared_context "fixture moabs in db" do
+  before(:all) do
+    setup
+    load_fixture_moabs
+  end
+  after(:all) do
+    # TODO: danger - if there are objects in the test db we want to keep
+    Settings.moab.storage_roots.each do |_name, storage_root|
+      storage_dir = File.join(storage_root, Settings.moab.storage_trunk)
+      PreservationCopy.where(endpoint_id: Endpoint.find_by(storage_location: storage_dir).id).each do |pc|
```

Comment:
likewise with the `@storage_dir_to_endpoint_id` hash instead of `Endpoint.find_by`?

Reply:
no access to instance vars in before/after(:all)